docs/cluster-workflow-templates.md

```diff
@@ -0,0 +1,162 @@
+# Cluster Workflow Templates
+
+> v2.8 and after
+
+## Introduction
+
+`ClusterWorkflowTemplates` are cluster scoped `WorkflowTemplates`. `ClusterWorkflowTemplate` 
+can be created cluster scoped like `ClusterRole` and can be accessed all namespaces in the cluster. 
+
+`WorkflowTemplate` documentation [link](./workflow-template.md)
+
+## Defining `ClusterWorkflowTemplate`
```

Comment:
One example here should be enough. Don't want to add too much redundant information